packages/chakra-ui/src/range-field/RangeField.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  Slider,
+  SliderFilledTrack,
+  SliderProps,
+  SliderThumb,
+  SliderTrack,
+} from "@chakra-ui/react";
+import { NumberFieldProps, useNumberFieldProps } from "@form-atoms/field";
+import { useFieldActions, useFieldInitialValue } from "form-atoms";
+
+import { ChakraField } from "../chakra-field";
+
+export const RangeField = ({
+  field,
+  label,
+  defaultValue,
+  ...sliderProps
+}: NumberFieldProps & SliderProps) => {
+  const props = useNumberFieldProps(field);
+  const actions = useFieldActions(field);
+
+  useFieldInitialValue(field, defaultValue);
+
+  return (
+    <ChakraField field={field} label={label}>
+      <Slider
+        name={props.name}
+        value={props.value || 0}
```

Comment:
```suggestion
        value={props.value}
```

do we really want to fallback to 0? does the component complain?

In flowbite, with undefined value the slider is nicely in the center, so it is ok for user to move it to one or other side.

could be consistent if possible

Reply:
Hmm, it was in the center but looked strange to me, but if it is a default behavior I will remove the fallback value to stay consistent across the codebase.

Reply:
yearh, range input starts in the middle

https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_range